.github/workflows/ci.yml

```diff
@@ -7,40 +7,115 @@ on:
   pull_request:
 
 jobs:
-  test:
+  initial-code-check:
```

Comment:
This one is so lightweight that I think it can be merged back e.g. to `code-style-check`